rotkehlchen/chain/zksync_lite/manager.py

```diff
@@ -199,15 +201,6 @@ def _query_and_save_transactions_for_range(
                     write_cursor=write_cursor,
                     transactions=unique_transactions,
                 )
-                if direction == 'older':
-                    current_start_ts = new_transactions[-1].timestamp
-                else:  # direction -> newer
-                    current_end_ts = new_transactions[-1].timestamp
-                ranges.update_used_query_range(
```

Comment:
Why remove this? Shouldn't something be saved here?

Reply:
Removed the used query range entirely as [discussed in discord](https://discord.com/channels/657906918408585217/1080783409640648765/1286788014533771274)